packages/share/example/lib/main.dart

```diff
@@ -44,7 +44,11 @@ class DemoAppState extends State<DemoApp> {
                   child: const Text('Share'),
                   onPressed: text.isNotEmpty
                       ? () {
-                          share(text);
+                          share(text,
+                              title: "Share Plugin Demo",
+                              media:
+                                  "https://pbs.twimg.com/profile_images/760249570085314560/yCrkrbl3_400x400.jpg",
```

Comment:
Done

Reply:
Using placeholder image URL without an extension will not work, as we will not be able to find the file type. We need a full actual URL.

Reply:
Sure. What I meant was that something like `http://via.placeholder.com/350x150.png` might look less scary to someone wanting to try out the share plugin. It may be just me, but I don't think everybody would immediately recognize Twitter's image host url.